README.md

```diff
@@ -29,7 +29,7 @@
 
 # 📖 MediChecker <a name="about-project"></a>
 
```

Comment:
- [ ] On the homepage kindly implement a search or filter functionality that filters items on the page using a parameter like a name or any distinct property.

  ![image](https://user-images.githubusercontent.com/81468675/211568162-fd7b14ea-1a48-4140-a197-0297215a7e55.png)

Reply:
Thanks! It's done.